lib/odoobaselib/__init__.py

```diff
@@ -57,6 +58,15 @@ def addons_config():
                     for glob in addons:
                         logging.debug("Expanding glob %s", glob)
                         for addon in iglob(os.path.join(SRC_DIR, repo, glob)):
+                            if not os.path.isdir(addon):
+                                continue
+                            addon_files = set(os.listdir(addon))
+                            has_manifest = manifest_files & addon_files
+                            py_module = '__init__.py' in addon_files
```

Comment:
I think the `py_module` check can be misleading. What if a `setup` dir contains one of `__init__.py` files? `has_manifest` should be enough IMHO.

Reply:
Indeed, I just thought that both conditions should be satisfied just like a regular Odoo module, though manifest should be ok. I did however check for both forms openerp and manifest so it keeps compatibility with V8.